README.adoc

```diff
@@ -0,0 +1,97 @@
+= Interrupt policy
+
+ifdef::env-github[]
+image:https://img.shields.io/static/v1?label=Available%20at&message=Gravitee.io&color=1EC9D2["Gravitee.io", link="https://download.gravitee.io/#graviteeio-apim/plugins/policies/gravitee-policy-interrupt/"]
+image:https://img.shields.io/badge/License-Apache%202.0-blue.svg["License", link="https://github.com/gravitee-io/gravitee-policy-interrupt/blob/master/LICENSE.txt"]
+image:https://img.shields.io/badge/semantic--release-conventional%20commits-e10079?logo=semantic-release["Releases", link="https://github.com/gravitee-io/gravitee-policy-interrupt/releases"]
+image:https://circleci.com/gh/gravitee-io/gravitee-policy-interrupt.svg?style=svg["CircleCI", link="https://circleci.com/gh/gravitee-io/gravitee-policy-interrupt"]
+endif::[]
+
+== Phase
+
+[cols="4*", options="header"]
+|===
+^|onRequest
+^|onResponse
+^|onRequestContent
+^|onResponseContent
+
+^.^| X
+^.^|
+^.^| X
+^.^|
+
+|===
+
+== Description
+
+`Interrupt` policy can be used to break the entire request processing in case of a condition (to define on the policy.
+By default, if no policy condition is defined, the policy will always break request processing).
```

Comment:
I don't see any "condition" in the source code. Something is missing or the doc is not accurate ?

Reply:
it is a policy condition, so something managed outside the policy itself

Reply:
ohhh right ! So if no condition, the policy will always break the request. And if there is a condition on the policy which is not met, then the policy will just not be executed.
Got it !

Reply:
Exactly !